Assets/00_Content/Scripts/UI/HUD.cs

```diff
@@ -19,12 +31,28 @@ public class HUD : MonoBehaviour {
 	}
 
 	void Update() {
-		if (RoundController.Instance != null)
-			roundTimerText.text = Mathf.RoundToInt(RoundController.Instance.RoundTimer).ToString();
+		UpdateClock();
+	}
+
+	private void UpdateClock() {
+		if (RoundController.Instance != null) {
+			float progress = RoundController.Instance.RoundTimer / RoundController.Instance.RoundDuration;
+
+			clockBackgroundDark.fillAmount = 1 - progress;
```

Comment:
`fillAmount` probably does this internally, but seeing as you removed the clamping on `RoundTimer` you might want to do that here.